packages/html/ds/src/checkbox/checkboxes-group.html

```diff
@@ -0,0 +1,63 @@
+{% from 'checkbox/checkbox.html' import govieCheckbox %} 
+{% from 'checkbox/helpers.html' import getSizeClass %}
+{% from 'heading/heading.html' import govieHeading %}
+{% from 'hint-text/hint-text.html' import govieHintText %}
+{% from 'error-text/error-text.html' import govieErrorText %}
+
+{% macro govieCheckboxesGroup(props) %}
+  <div data-testid="govie-checkboxes" data-module="gieds-checkboxes" class="gi-flex">
+    {% if props.errorMessage %}
+    <div class="gi-w-5 gi-border-l-[5px] gi-border-l-red-600 gi-border-solid"></div>
+    {% endif %}
+    <fieldset>
+      <legend class="gi-mb-3.5 gi-font-primary sm:gi-text-sm md:gi-text-md lg:gi-text-lg">
+        {% if props.title.asHeading %}
+          {{ govieHeading({
+            "size": "lg",
+            "tag": "h1",
```

Comment:
We should probably also pass `size` and `tag` as options for the heading.

Reply:
good idea, I'll do that. This is what ogcio storybook has but yes, it's better to give users this flexibility

Reply:
Fixed